{ 'stand_up_front_100': { 'head': [ { 'duration': 1.0,
                                      'name': u'stand_up_front_0100',
                                      'positions': [-0.0001, -0.4936],
                                      'starttime': 0.0}],
                          'left_arm': [ { 'duration': 2.0,
                                          'name': u'stand_up_front_0100',
                                          'positions': [ -0.0658,
                                                         0.9669,
                                                         0.0002,
                                                         0.0008,
                                                         -0.0171,
                                                         0.0008,
                                                         0.0008],
                                          'starttime': 0.0},
                                        { 'duration': 0.992,
                                          'name': u'stand_up_front_0200',
                                          'positions': [ 0.0981,
                                                         0.9654,
                                                         -0.0,
                                                         3.1062,
                                                         0.0224,
                                                         -0.0039,
                                                         -0.0039],
                                          'starttime': 2.0},
                                        { 'duration': 2.008,
                                          'name': u'stand_up_front_0300',
                                          'positions': [ -1.3568,
                                                         0.9896,
                                                         0.0,
                                                         3.1416,
                                                         -1.8865,
                                                         0.0003,
                                                         0.0005],
                                          'starttime': 2.992},
                                        { 'duration': 1.999,
                                          'name': u'stand_up_front_0400',
                                          'positions': [ -1.4334,
                                                         -0.4407,
                                                         -0.0003,
                                                         3.1402,
                                                         -1.8885,
                                                         -0.0081,
                                                         -0.0079],
                                          'starttime': 5.0},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_0500',
                                          'positions': [ -1.3551,
                                                         -0.4196,
                                                         0.0023,
                                                         3.1408,
                                                         -1.8855,
                                                         1.0135,
                                                         -1.9965],
                                          'starttime': 6.9990000000000006}],
                          'left_leg': [ { 'duration': 0.974,
                                          'name': u'stand_up_front_0100',
                                          'positions': [ 0.0013,
                                                         -0.1055,
                                                         -0.0018,
                                                         0.9948,
                                                         0.0002,
                                                         -0.0001],
                                          'starttime': 2.003},
                                        { 'duration': 1.0,
                                          'name': u'stand_up_front_0200',
                                          'positions': [ 0.0012,
                                                         -0.1058,
                                                         -0.0032,
                                                         0.995,
                                                         1.8005,
                                                         -0.0001],
                                          'starttime': 2.984},
                                        { 'duration': 1.013,
                                          'name': u'stand_up_front_0300',
                                          'positions': [ 0.0009,
                                                         -0.1082,
                                                         -0.0102,
                                                         0.3855,
                                                         1.8003,
                                                         0.0],
                                          'starttime': 3.984}],
                          'right_arm': [ { 'duration': 2.0,
                                           'name': u'stand_up_front_0100',
                                           'positions': [ 0.0658,
                                                          -0.9669,
                                                          -0.0002,
                                                          -0.0008,
                                                          0.0171,
                                                          -0.0008,
                                                          -0.0008],
                                           'starttime': 0.0},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_0200',
                                           'positions': [ -0.0981,
                                                          -0.9654,
                                                          0.0,
                                                          -3.1062,
                                                          -0.0224,
                                                          0.0039,
                                                          0.0039],
                                           'starttime': 2.0},
                                         { 'duration': 1.998,
                                           'name': u'stand_up_front_0300',
                                           'positions': [ 1.3568,
                                                          -0.9896,
                                                          -0.0,
                                                          -3.1416,
                                                          1.8865,
                                                          -0.0003,
                                                          -0.0005],
                                           'starttime': 3.0},
                                         { 'duration': 2.0,
                                           'name': u'stand_up_front_0400',
                                           'positions': [ 1.4334,
                                                          0.4407,
                                                          0.0003,
                                                          -3.1402,
                                                          1.8885,
                                                          0.0081,
                                                          0.0079],
                                           'starttime': 4.998},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_0500',
                                           'positions': [ 1.3551,
                                                          0.4196,
                                                          -0.0023,
                                                          -3.1408,
                                                          1.8855,
                                                          -1.0135,
                                                          1.9965],
                                           'starttime': 6.998}],
                          'right_leg': [ { 'duration': 0.99,
                                           'name': u'stand_up_front_0100',
                                           'positions': [ -0.0013,
                                                          0.1055,
                                                          0.0018,
                                                          -0.9948,
                                                          -0.0002,
                                                          0.0001],
                                           'starttime': 2.006},
                                         { 'duration': 1.01,
                                           'name': u'stand_up_front_0200',
                                           'positions': [ -0.0012,
                                                          0.1058,
                                                          0.0032,
                                                          -0.995,
                                                          -1.8005,
                                                          0.0001],
                                           'starttime': 2.9959999999999996},
                                         { 'duration': 1.0,
                                           'name': u'stand_up_front_0300',
                                           'positions': [ -0.0009,
                                                          0.1082,
                                                          0.0102,
                                                          -0.3855,
                                                          -1.8003,
                                                          -0.0],
                                           'starttime': 4.006}],
                          'torso': [ { 'duration': 1.0,
                                       'name': u'stand_up_front_0100',
                                       'positions': [0.0001, 0.0019],
                                       'starttime': 1.002}]}}
